### PR TITLE
Docs/demo

### PR DESCRIPTION
**Pull request by ZengBeauty**
## Description

The RN side uses the same code as the WEB side, and modifies the repository directory.

## Type of change

Please select the relevant option.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] This modification changed the repository directory structure

### CODE REVIEW

**Review by youngjuning — APPROVED** [review]
LGTM~

File: .github/workflows/gh-pages.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: github pages
+on:
+  push:
+    branches:
+      - docs/demo # default branch
```
Comment (ShinyLeee) [inline .github/workflows/gh-pages.yml:5]: 主分支后面得调整吧？

Comment (youngjuning) [reply .github/workflows/gh-pages.yml:5]: 这个合了之后改比较合适

File: packages/tuya-panel-theme/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "scripts": {
+    "prebuild": "rimraf lib/*",
+    "build": "tsc -d",
+    "dev": "tsc -w"
+  },
+  "bugs": {
+    "url": "https://github.com/tuya/tuya-panel-kit/issues"
+  },
+  "devDependencies": {
+    "tuya-panel-utils": "^0.0.0"
```
Comment (ShinyLeee) [inline packages/tuya-panel-theme/package.json:32]: 为什么是 devDependencies？

File: packages/tuya-panel-utils/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "build": "tsc -d",
+    "dev": "tsc -w",
+    "test": "NODE_ENV=testing TZ=UTC jest -u --silent --forceExit",
+    "test:watch": "jest --watch"
+  },
+  "bugs": {
+    "url": "https://github.com/tuya/tuya-panel-kit/issues"
+  },
+  "peerDependencies": {
+    "react": "16.8.3",
```
Comment (ShinyLeee) [inline packages/tuya-panel-utils/package.json:34]: 工具库是纯粹的，没有依赖这些东西吧

**Review by ShinyLeee — APPROVED** [review]

File: packages/tuya-panel-utils/package.json (hunk 3 above)
Comment (ZengBeauty) [reply packages/tuya-panel-utils/package.json:34]: 除了svg 其他的依赖了，已删除 svg

**Pull request merged**